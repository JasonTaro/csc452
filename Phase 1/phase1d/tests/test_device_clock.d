test_device_clock.o: tests/test_device_clock.c ../phase1.h \
  /Users/kyle/include/usloss.h /Users/kyle/include/usyscall.h
