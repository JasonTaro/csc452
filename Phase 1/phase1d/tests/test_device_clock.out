 
---------Starting Test CLOCK ----------
wait period : 0
wait period : 0
wait period : 0
wait period : 0
wait period : 0
wait period : 0
wait period : 0
wait period : 0
wait period : 0
wait period : 0
 ---------Ending Test CLOCK ----------
Sentinel quitting.
No runnable processes, halting.
